doc/admin-guide/files/sni.yaml.en.rst

```diff
@@ -119,6 +119,11 @@ forward_route             Destination as an FQDN and port, separated by a colon
                           This is similar to tunnel_route, but it terminates the TLS connection and forwards the
                           decrypted traffic. |TS| will not interpret the decrypted data, so the contents do not
                           need to be HTTP.
+
+partial_blind_route       Destination as an FQDN and port, separated by a colon ``:``.
+
+                          This is similar to forward_route, but it terminates the upstream connection.  |TS| will
+                          not interpret the decrypted data, so the contents do not need to be HTTP.
```

Comment:
The description could be improved.  Perhaps with an example.  By looking through the PR, I think what you are doing is terminating on the ATS box, creating another TLS connection upstream, and blindly copying the bytes between the two.  Is that correct?

Reply:
Correct! Also, if you have a better name for this feature, suggestions are definitely welcome.

Reply:
I'm not so good with the naming thing either.  May be in the description you can say

This is similar to forward_route in that |TS| terminates the incoming TLS connection.  In addition partial_blind_route creates a new TLS connection to the specified origin.  It does not interpret the decrypted data before passing it to the origin TLS connection, so the contents do not need to be HTTP.

Reply:
Thanks! I've updated the wording.